src/compiler/checker.ts

```diff
@@ -13473,31 +13473,32 @@ namespace ts {
                         node.parent.openingElement.attributes :
                         undefined; // node.parent is JsxFragment with no attributes
 
-            if (jsxAttributes) {
-                // When we trying to resolve JsxOpeningLikeElement as a stateless function element, we will already give its attributes a contextual type
-                // which is a type of the parameter of the signature we are trying out.
-                // If there is no contextual type (e.g. we are trying to resolve stateful component), get attributes type from resolving element's tagName
-                const attributesType = getContextualType(jsxAttributes);
+            if (!jsxAttributes) {
+                return anyType; // don't check children of a fragment

```

Comment:
It's better to return `undefined` than `anyType` here because it will cause the calling code to bail out earlier

Reply:
Changed.